src/request-pipeline/context/index.ts

```diff
@@ -426,6 +426,11 @@ export default class RequestPipelineContext extends BaseRequestPipelineContext {
         this.goToNextStage = false;
     }
 
+    closeConnectionOnError (): void {
+        if (this.req.socket)
```

Comment:
We discussed in private and decided to add `req.destroy()` calling in case of some changes in future node js version